Include/internal/pycore_time.h

```diff
@@ -305,41 +306,6 @@ PyAPI_FUNC(PyTime_t) _PyDeadline_Init(PyTime_t timeout);
 PyAPI_FUNC(PyTime_t) _PyDeadline_Get(PyTime_t deadline);
 
 
-// --- _PyTimeFraction -------------------------------------------------------
-
-typedef struct {
-    PyTime_t numer;
-    PyTime_t denom;
-} _PyTimeFraction;
-
-// Set a fraction.
-// Return 0 on success.
-// Return -1 if the fraction is invalid.
-extern int _PyTimeFraction_Set(
```

Comment:
Did you move these 3 _PyTimeFraction functions on purpose?

Reply:
No, by accident. I've put them back.